docs/source/experiments/regression-robust04.md

```diff
@@ -0,0 +1,86 @@
+# PISA: Regression Tests for [Disks 4 & 5](https://trec.nist.gov/data_disks.html) (Robust04)
+
+## Indexing
+
+First, we will create a directory where all the indexes are going to be stored:
+
+```
+mkdir robust04
+```
+
+### Parsing
+
+```
+gzip -dc $(find /path/to/disk45/ -type f -name '*.*z' \( -path '*/disk4/fr94/[0-9]*/*' -o -path '*/disk4/ft/ft*' -o -path '*/disk5/fbis/fb*' -o -path '*/disk5/latimes/la*' \)) | bin/parse_collection -f trectext -b 10000 --stemmer porter2 --content-parser html -o robust04/fwd
+```
+
+You can replace `gzip -dc` with `zcat` on Linux or `gzcat` on MacOS.
+The directory `/path/to/disk45/` should be the root directory of [TREC Disks 4 & 5](https://trec.nist.gov/data_disks.html).
+
+### Inverting
+
+```
+/path/to/pisa/build/bin/invert -i robust04/fwd -o robust04/inv -b 400000 --term-count $(cat robust04/fwd.terms | wc -l)
```

Comment:
The `--term-count` is now optional. If missing, it will try to read the term lexicon size, which in this scenario is produced by `parse_collection`, so maybe we should keep it simple and remove it?